local-volume/README.md

```diff
@@ -89,14 +142,22 @@ done
 ```
 
 2. Run the local cluster.
+
 ```console
-$ ALLOW_PRIVILEGED=true LOG_LEVEL=5 FEATURE_GATES=PersistentLocalVolumes=true hack/local-up-cluster.sh
+$ ALLOW_PRIVILEGED=true LOG_LEVEL=5 FEATURE_GATES=$KUBE_FEATURE_GATES hack/local-up-cluster.sh
 ```
 
-3. Continue with [Creating local persistent volumes](#creating-local-persistent-volumes)
-   below.
+### Step 2: Creating a StorageClass (1.9+)
+To delay volume binding until pod scheduling, a StorageClass needs to be created
```

Comment:
Should we mention that this storage class parameter is required to enable scheduling of multiple local PVs on the same pod?